src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -62,12 +61,13 @@
     private static final int DEFAULT_EMBEDDED_HTTP_SERVER_PORT = 8080;
 
     @SuppressWarnings({"checkstyle:NPathComplexity"})
+    @SuppressFBWarnings("REC_CATCH_EXCEPTION")
```

Comment:
> If we do not know what and why are we catching, we should probably remove the whole try-catch instead of suppressing the warning.

And have main() throw Exception?

Reply:
To have `main()` throw what exception? The error suggests there is no exception being thrown other than possible RuntimeException which can for sure be thrown out of `main()`.

Reply:
Then the error is wrong, since there are 3 different non-runtime exceptions being thrown.
Hence probably why we had that all-catch in the first place.

Reply:
Then maybe what you had there was right, but you didn't explained it netiehr in the PR comments nor in the code comments.

Reply:
I doubt I can explain it better then the error's original docs, from the link posted above, and this one:
* https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rec-exception-is-caught-when-exception-is-not-thrown-rec-catch-exception

But I fail to see why this approach is better in our case (or for that matter in all cases ...).

Reply:
I guess I need to look at the actual code.

Reply:
Ok, so, the code throws 3 different exceptions:
* `IOException`
* `MalformedObjectNameException`
* `ParseException`

So the error tells you to do either this:

```java
        } catch (MalformedObjectNameException|IOException|ParseException ex) {
            log.error("Error starting the bridge", ex);
            System.exit(1);
        }
```

eventually something like this if we want to catch the RuntimeException as well:

```java
        } catch (RuntimeException|MalformedObjectNameException|IOException|ParseException ex) {
            log.error("Error starting the bridge", ex);
            System.exit(1);
        }
```

So why not use that? This is what it suggests.